provider/user_data.go

```diff
@@ -0,0 +1,72 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/firehydrant/terraform-provider-firehydrant/firehydrant"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceUser() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataFireHydrantPriority,
```

Comment:
This will need to be `dataFireHydrantUser`
```suggestion
		ReadContext: dataFireHydrantUser,
```